src/_blog/metrics-in-go-libp2p.md

```diff
@@ -0,0 +1,184 @@
+---
+tags:
+- metrics
+- prometheus
+- grafana
+title: Metrics in go-libp2p
+description:
+date: 2023-06-15
```

Comment:
Do we need to update this?

Reply:
We can update it once we are ready to release. https://github.com/libp2p/blog/pull/77#pullrequestreview-1505832300

Reply:
done